exporter.py

```diff
@@ -0,0 +1,105 @@
+import logging
+import time
+from os import getenv
+
+import os
+
+import urllib3
+from prometheus_client import REGISTRY, start_http_server
+
+from collectors.PyVimServiceCollector import PyVimServiceCollector
+from collectors.SshServiceCollector import SshServiceCollector
+from collectors.EsxiOverallStateCollector import EsxiOnlineStateCollector
+from modules.Exceptions import VCenterException
+
+# init logger once
+logger = logger = logging.getLogger('esxi-exporter')
+logger.setLevel(logging.DEBUG)

```

Comment:
Please implement an `OptionParser` to set this on demand using CLI

Reply:
done.

Reply:
Option parser now allows `-d` option to enable debug output. But most (all) loggers use info, error, warning, critical but not debug.

Reply:
Haven't  had a look at them yet. Two things, add `-v` for `INFO` level.
Normal logs we want to see during runtime are `ERROR` and `WARNING`, nothing else, unless specified.

You might reconsider, what is logging info, maybe it is debug?

Reply:
Changed that. Now most things are debug. Some are info.